DemoApp-ObjC/DemoApp-ObjC.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm leaning towards adding this (and the workspace in general) to the `.gitignore`. I'm afraid of giving the impression that the pods are ready to go when a pub clones this repo; our demo should be as simple/clean as possible. @thomasmso @santoshbagadi  thoughts?

Reply:
Definitely should have it clean as possible, and the publishers should manually install the pods - good catch.